tests/suites/spaces_ec2/util.sh

```diff
@@ -0,0 +1,99 @@
+ec2_check_prerequisites_for_space_tests() {
```

Comment:
Considering we're in `spaces_ec2` can we remove the stutter?

```suggestion
check_prerequisites() {
```